Microsoft/WebTemplateStudio
# Summary page show licenses

Task:https://microsoftgarage.visualstudio.com/Intern%20GitHub/_sprints/taskboard/WTS%20Team/Intern%20GitHub/Y19-VS/06
Task:https://microsoftgarage.visualstudio.com/Intern%20GitHub/_sprints/taskboard/WTS%20Team/Intern%20GitHub/Y19-VS/07

This change is to make current summary page (middle panel) as the expected screenshot. Changes included in my code:
- update summary page into show less information
- add view licenses modal
- able to resize the modals based on modal type ( so far only _post gen modal_ and _view licenses modal_ have width 30%, the others have width 40%)

**How to test:**
1. generate any project type with any page
2. at summary page, clicks on **View Licenses** button, it opens a modal and contains all the corresponding licenses
3. at summary page, clicks on **Give Feedback** button, it should redirect you to GitHub Issue page

**Expected from my PR:**
![licensesmodal](https://user-images.githubusercontent.com/12874317/60615608-65c27f00-9d84-11e9-9e0b-78fe1ccaa676.PNG)

**New Summary Page from Designer:** ( the background image in this design is from the internet)
![summary page](https://user-images.githubusercontent.com/12874317/60305807-b5bbc480-98f2-11e9-8f98-7eea2e55d991.png)
